Dockerfile.ubuntu

```diff
@@ -1,6 +1,10 @@
 FROM ubuntu:16.04
+ARG docker_ver=17.12.0
 
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+# links to commit hashes are listed inside posted Dockerfiles https://hub.docker.com/r/library/docker/
+# NOTE: must match engine version that is directly pulled from Alpine's Dockerfile
+ENV DIND_COMMIT 3b5fac462d21ca164b3778647420016315289034
```

Comment:
These instructions took me too long to figure out. Can we use the expanded notes I added in my PR (assuming they're correct)? https://github.com/mesosphere/dcos-jenkins-dind-agent/pull/24/files
```
# links to commit hashes are listed inside posted Dockerfiles https://hub.docker.com/r/library/docker/ 
# NOTE: must match engine version that is directly pulled from Alpine's Dockerfile
# - go to https://hub.docker.com/r/library/docker/
# - click on the matching alpine version tag (eg, 17.12.0-dind)
# - pull the DIND_COMMIT has from the Dockerfile that opens, for 17.12.0-dind it will be:
#   https://github.com/docker-library/docker/blob/de9fda490429cf83734ef78b58f0ae9cfed1b087/17.12/dind/Dockerfile
```

Reply:
I will add this.